obspy/imaging/tests/test_ray_paths.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import unittest
+
+import os
+
+from obspy.core.util.testing import ImageComparison
+from obspy import read_inventory, read_events
+from obspy.imaging.ray_paths import plot_rays, get_ray_paths
+
+
+try:
+    from mayavi import mlab  # @UnusedImport # NOQA
+    HAS_MAYAVI = True
+except ImportError:
+    HAS_MAYAVI = False
+
+
+class PathPlottingTestCase(unittest.TestCase):
+    """
+    Test suite for obspy.core.util.geodetics
+    """
+    def setUp(self):
+        # load an inventory and an event catalog to test
+        # the ray path routines. Careful, the full catalog
+        # test is quite long and is therefore commented out
+        # by default
+        self.path = os.path.join(os.path.dirname(__file__), 'images')
+        pass
+
+    def test_compute_ray_paths(self):
+        # careful, the full inventory, catalog test is long (1min)
+        # greatcircles = get_ray_paths(
+        #        inventory=self.inventory, catalog=self.catalog,
+        #        phase_list=['P'], coordinate_system='XYZ',
+        #        taup_model='iasp91')
```

Comment:
yes. I wanted to make a nice image for the gallery, but if the test doesn't work on travis anyway because of ancient OpenGL, it is not going to work...